### DISCUSSION TITLE
Cambio router  e mancanza di accesso in locale

### DISCUSSION THREAD

**Discussion by Starc73** [indomus/forum#3153] [HUB: Home Assistant]
Ho appena installato HASOS su un NUC, per l'installazione ho utilizzato il nuc collegato con cavo di rete al router di casa e il server era raggiungibile in rete locale da un altro pc. Ho deciso di collegarlo ad un router diverso con sim dati e dopo aver riavviato il nuc e Hassos non riuscivo più a raggiungere in locale da un alto  pc il server. 

Come posso risolvere ?

Grazie

**Comment by indomus** [comment]
puntando al nuovo indirizzo, almeno il nuovo ip dovrebbe risponderti (ovviamente l'host dal quale ti colleghi ad esso dev'essere sulla stessa rete o ruotato verso essa)

**Comment by Starc73** [comment]
Ma scusa essendo sulla stessa rete non dovrebbero in entrambi i casi essere accessibili all'indirizzo http://homeassistant.local:8123/ ? Ho provato anche a ricollegare il NUC al router di casa ma niente, non è più possibile accedere al server

**Reply by indomus** [reply]
perché dovrebbero essere sulla stessa rete, se hai cambiato router?

**Reply by Starc73** [reply]
Nel primo caso il nuc è collegato al router di casa con cavo LAN mentre il pc portatile con cui mi collego è collegato in wifi al router di casa. Nel secondo caso  il nuc è collegato al router con sim con cavo LAN mentre il pc portatile con cui mi collego è collegato in wifi a router sim

**Reply by indomus** [reply]
ok. allora i casi sono:

- l'host non ha ottenuto l'ip dal dhcp server del router;
- mdns non sta funzionando

nel primo caso devi capire perché. nel secondo, prova a puntare su http://IP_DEL_SERVER:8123, vedrai che ti risponderà.
